lib/Client.js

```diff
@@ -218,23 +211,19 @@ class Client extends EventEmitter {
         this.lastConnect = 0;
         this.channelGuildMap = {};
         this.threadGuildMap = {};
-        this.groupChannels = new Collection(GroupChannel);
+        this.shards = new ShardManager(this);
```

Comment:
duplicate shard manager declaration

Reply:
what I thought I removed that during the merge conflict